src/models/user.model.js

```diff
@@ -0,0 +1,112 @@
+const dir = require('../helpers/dirConfig');
+const db = require(dir.libs + '/db');
+
+class UserModel {
+    /**
+     * @description
+     * Fetch all users from database.
+     * @returns {Promise<object[]>} A promise that resolves to an array of user objects.
+     */
+    async getAllUsers() {
+        const query = await db('users');
```

Comment:
лишнее объявление переменной можно просто вернуть:
```suggestion
        return await db('users');
```